### PR TITLE
Fix issue with commented out feature files

### PR DESCRIPTION
**Pull request by jmezach**
When a feature file is commented out in its entirety this resulted in a NullReferenceException causing a nasty crash of the Augurk CLI. We now handle this situation more gracefully.

### CODE REVIEW

File: src/Augurk.CommandLine/Commands/PublishCommand.cs [hunk 1]
```diff
@@ -113,6 +113,11 @@ private int PublishFeatureFiles()
                             return -1;
                         }
                     }
+                    catch (InvalidOperationException)
+                    {
+                        Console.Error.WriteLine($"Unable to parse feature file '{featureFile}'. Possible causes are that the feature file is commented out entirely.");
```
Comment (marktaling) [inline src/Augurk.CommandLine/Commands/PublishCommand.cs]: Should we make this assumption? Or should we just mention the file does not contain any Gherkin content?

File: src/Augurk.CommandLine/Commands/PublishCommand.cs [hunk 2]
```diff
@@ -113,6 +113,11 @@ private int PublishFeatureFiles()
...
+                        return -1;
```
Comment (marktaling) [inline src/Augurk.CommandLine/Commands/PublishCommand.cs]: Return when a single file has no valid content?
Shouldn't we at least provide the option to skip such files?

**Review by marktaling — COMMENTED** [review]
Not sure which way to go here... but I imagine it might happen that a single file is commented out while all others are just valid features.
As in a lot of cases people don't allow Augurk to fail your build (as we've seen), wouldn't we at least want the option to publish everything that *is* valid? Or perhaps even make that the default?

File: src/Augurk.CommandLine/Commands/PublishCommand.cs (hunk 1 above)
Comment (jmezach) [reply src/Augurk.CommandLine/Commands/PublishCommand.cs]: ;) Fair enough

File: src/Augurk.CommandLine/Commands/PublishCommand.cs (hunk 2 above)
Comment (jmezach) [reply src/Augurk.CommandLine/Commands/PublishCommand.cs]: Skipping them for now, but writing a warning, which should be picked up by the build pipeline (hopefully)

**Pull request merged**